metrictank.go

```diff
@@ -57,7 +58,10 @@ var (
 	accountingPeriodStr = flag.String("accounting-period", "5min", "accounting period to track per-org usage metrics")
 
 	// Clustering:
-	primaryNode = flag.Bool("primary-node", false, "the primary node writes data to cassandra. There should only be 1 primary node per cluster of nodes.")
+	primaryNode      = flag.Bool("primary-node", false, "the primary node writes data to cassandra. There should only be 1 primary node per cluster of nodes.")
+	peersStr         = flag.String("peers", "", "http/s addresses of other nodes, comma separated. use this if you shard your data and want to query other instances")
+	probeIntervalStr = flag.String("probe-interval", "2s", "Interval to probe peer nodes")
+	mode             = flag.String("mode", "single", "Operating mode of cluster. (single|multi)")
```

Comment:
The "mode" flag is only used when deciding on whether other nodes need to be used to handle graphite queries.  I wasnt sure if there was a use case where you would want to know about other nodes in the cluster but not co-ordinate queries with them. The example i had in mind is if we later support automatic primary failover for a cluster of nodes that all handle all partitions.